src/components/player/ShowAnswer.tsx

```diff
@@ -1,16 +1,19 @@
 import { state$ } from "../../state";
 
-function ShowAnswer() {    
-    
-    // wait 5 sec
-    // emit WS(waitingforquiz)
-    // navigate to WaitingForNextQuestion
-
-    return (
-        <div>
-            {state$.quiz.isAnswerCorrect.get() ? 'yay you got it right' : 'boo wrong'}
+function ShowAnswer() {
+  return (
```

Comment:
can you clean this up by doing something like this:
```js
function ShowAnswer()  {
    if (state$.quiz.isAnswerCorrect.get()) {
        return <CorrectComponent />
    } else {
        return <WrongComponent />
    }
}```
you don't need to make a correct and wrong component, i just used those as a placeholder to show you the structure of what i am talking about.

Reply:
to make it full-screen, try making the element's position `absolute` and with `top: 0; left: 0`